cpp/examples/abm_minimal.cpp

```diff
@@ -117,25 +92,33 @@ int main()
     world.get_individualized_location(school).get_infection_parameters().set<mio::abm::MaximumContacts>(20);
     // At every workplace, maximum contacts are 10.
     auto work = world.add_location(mio::abm::LocationType::Work);
-    world.get_individualized_location(work).get_infection_parameters().set<mio::abm::MaximumContacts>(10);
+    world.get_individualized_location(work).get_infection_parameters().set<mio::abm::MaximumContacts>(20);
+
+    // Increase aerosol transmission for all locations
+    world.parameters.get<mio::abm::AerosolTransmissionRates>() = 10.0;
+    // Increase contact rate for all people between 15 and 34 (i.e. people meet more often in the same location)
+    world.get_individualized_location(work)
+        .get_infection_parameters()
+        .get<mio::abm::ContactRates>()[{AGE_GROUP_15_TO_34, AGE_GROUP_15_TO_34}] = 10.0;
 
     // People can get tested at work (and do this with 0.5 probability) from time point 0 to day 30.
     auto testing_min_time      = mio::abm::days(1);
     auto probability           = 0.5;
     auto start_date            = mio::abm::TimePoint(0);
-    auto end_date              = mio::abm::TimePoint(0) + mio::abm::days(30);
+    auto end_date              = mio::abm::TimePoint(0) + mio::abm::days(10);
     auto test_type             = mio::abm::AntigenTest();
     auto testing_criteria_work = mio::abm::TestingCriteria();
     auto testing_scheme_work =
         mio::abm::TestingScheme(testing_criteria_work, testing_min_time, start_date, end_date, test_type, probability);
     world.get_testing_strategy().add_testing_scheme(mio::abm::LocationType::Work, testing_scheme_work);
 
     // Assign infection state to each person.
-    // The infection states are chosen randomly.
-    auto persons = world.get_persons();
-    for (auto& person : persons) {
+    // The infection states are chosen randomly with the following distribution
+    std::vector<double> population_distribution{0.5, 0.3, 0.05, 0.05, 0.05, 0.05, 0.0, 0.0};
```

Comment:
I would change the name to `infection_distribution` to avoid confusion.

Reply:
yes, thank you!